lessons/editing-audio-with-audacity.md

```diff
@@ -26,45 +26,39 @@ review-ticket: 15
 
 For those interested in audio, basic sound editing skills go a long way. Being able to handle and manipulate the materials can help you take control of your object of study: you can zoom in and extract particular moments to analyze, process the audio, and upload the materials to a server to compliment a blog post on the topic. On a more practical level, these skills could also allow you to record and package recordings of yourself or others for distribution. That guest lecture taking place in your department? Record it and edit it yourself! Doing so is a lightweight way to distribute resources among various institutions, and it also helps make the materials more accessible for readers and listeners with a wide variety of learning needs.
 
-In this lesson you will learn how to use *[Audacity](http://audacityteam.org/)* to load, record, edit, mix, and export audio files. Sound editing platforms are often expensive and offer extensive capabilities that can be overwhelming to the first-time user, but *Audacity* is a free and open source alternative that offers powerful capabilities for sound editing with a low barrier for entry. 
+In this lesson you will learn how to use *[Audacity](http://audacityteam.org/)* to load, record, edit, mix, and export audio files. Sound editing platforms are often expensive and offer extensive capabilities that can be overwhelming to the first-time user, but *Audacity* is a free and open source alternative that offers powerful capabilities for sound editing with a low barrier for entry.
 
-For this lesson we will work with two audio files: a recording of [Bach's Goldberg Variations](https://musopen.org/music/download/6270/) available from *[MusOpen](https://musopen.org/)* and another recording of your own voice that will be made in the course of the lesson. 
+For this lesson we will work with two audio files: a recording of <a href="{{ root_url }}/assets/editing-audio-with-audacity/bach-goldberg-variations.mp3">Bach's Goldberg Variations</a> and another recording of your own voice that will be made in the course of the lesson.
```

Comment:
I think you can just use the notation `[Bach's Goldberg Variations]({{ root_url }}...)` instead of the full HTML, can't you?

Reply:
Totally. Done.